### PR TITLE
Moving assets from oci-emea-big-compute 

### PR DESCRIPTION
**Pull request by adelmonaco-oci**
As per subject.

### CODE REVIEW

File: cloud-infrastructure/ai-infra-gpu/Benchmarks/gds-benchmarks/README.md
Comment (AlexanderHodicke) [inline cloud-infrastructure/ai-infra-gpu/Benchmarks/gds-benchmarks/README.md:1]: Spell checking

File: cloud-infrastructure/ai-infra-gpu/Benchmarks/gds-benchmarks/scripts/2csv.sh
Comment (AlexanderHodicke) [inline cloud-infrastructure/ai-infra-gpu/Benchmarks/gds-benchmarks/scripts/2csv.sh:1]: @oheimburger Could you please have a look

File: cloud-infrastructure/ai-infra-gpu/Benchmarks/iozone-benchmark/rpms/iozone-3.424-2.el7.rf.x86_64.rpm
Comment (AlexanderHodicke) [inline cloud-infrastructure/ai-infra-gpu/Benchmarks/iozone-benchmark/rpms/iozone-3.424-2.el7.rf.x86_64.rpm:1]: Copyright (c) 2001 Yves Rougy Yves@Rougy.net inside the zip and code?

File: cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_client/connauthfile
Comment (AlexanderHodicke) [inline cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_client/connauthfile:1]: Connauthfile? Security? Binay?

File: cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_server/connauthfile
Comment (AlexanderHodicke) [inline cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_server/connauthfile:1]: Same as before

File: cloud-infrastructure/ai-infra-gpu/Benchmarks/gds-benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+
+Each test will be logged under the corresponding directory. The naming of the log files has the following format: ``gdsio_s<block size>_m<mode number>_w<threads / workers>_r<repetition number>_d<timestamp>.log``  
+gdsio_s1M_m2_w1_r1_d1671836744.log  
+For example, this log refers to a test that used 1M as block size, Mode 2 (CPU_GPU), 1 thread, and its timestamp.  
+Each log also contains the full command that was issued to run:  
+```
+/usr/local/cuda/gds/tools/gdsio -D /mnt/nvme0/ -d 0 -w 1 -s 8G -x 2 -i 1M -I 1 -V >> /home/ubuntu/gds_benchmarks/scripts/../logs/CPU_GPU//gdsio_s1M_m2_w1_r1_d1671836744.log
```
Comment (oheimburger) [inline cloud-infrastructure/ai-infra-gpu/Benchmarks/gds-benchmarks/README.md:39]: This expects a predefined user. Please don't expect any predefined user and make it more flexible by not using fixed usernames.

File: cloud-infrastructure/ai-infra-gpu/Benchmarks/gds-benchmarks/scripts/initialize.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+mdadm --create --verbose /dev/md0 --level=0 --raid-devices=4 /dev/nvme0n1 /dev/nvme1n1 /dev/nvme2n1 /dev/nvme3n1
+
+echo "Formatting RAID0 volume"
+mkfs.ext4 /dev/md0 
+
+echo "Mounting device md0"
+mount -o data=ordered /dev/md0 /mnt/nvme0
+
+echo "Adjusting privileges"
+chown -R ubuntu. /mnt/nvme0/
```
Comment (oheimburger) [inline cloud-infrastructure/ai-infra-gpu/Benchmarks/gds-benchmarks/scripts/initialize.sh:27]: This expects a predefined user. Please don't expect any predefined user and make it more flexible by not using fixed usernames.

File: cloud-infrastructure/ai-infra-gpu/Benchmarks/iozone-benchmark/rpms/iozone-3.424-2.el7.rf.x86_64.rpm
Comment (oheimburger) [reply cloud-infrastructure/ai-infra-gpu/Benchmarks/iozone-benchmark/rpms/iozone-3.424-2.el7.rf.x86_64.rpm:1]: What is the license for this part?

File: cloud-infrastructure/ai-infra-gpu/Benchmarks/iozone-benchmark/scripts/initiate.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+mkfs.ext4 /dev/md0 
+
+echo "Creating mount dir"
+mkdir /mnt/data
+
+echo "Mounting device md0"
+mount -o data=ordered /dev/md0 /mnt/data
+
+echo "Adjusting privileges"
+chown -R opc. /mnt/data/
```
Comment (oheimburger) [inline cloud-infrastructure/ai-infra-gpu/Benchmarks/iozone-benchmark/scripts/initiate.sh:18]: This expects a predefined user. Please don't expect any predefined user and make it more flexible by not using fixed usernames.

File: cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_client/beegfs-client-autobuild.conf [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# This is a config file for the automatic build process of BeeGFS client kernel
+# modules.
+# http://www.beegfs.com
```
Comment (oheimburger) [inline cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_client/beegfs-client-autobuild.conf:3]: What is the license for beegfs?

File: cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_client/connauthfile
Comment (oheimburger) [reply cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_client/connauthfile:1]: Do not store any keys in github. PR should be removed.

File: cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_client/lib/start-stop-functions [hunk 5]
```diff
@@ -0,0 +1,688 @@
+#!/bin/bash
+
+# The original version of this file was taken from the CentOS5
+# initscripts-8.45.30-2.el5.x86_64 package and as this package is licensensed
+# under GPLv2 and GPLv2+ this entire file is provided under those licenses.
```
Comment (oheimburger) [inline cloud-infrastructure/ai-infra-gpu/Storage/beegfs-singlenode-setup/beegfs_client/lib/start-stop-functions:5]: This license requires that we use the same license for the whole solution. Please contact legal for a legal approval.

**Review by oheimburger — CHANGES_REQUESTED** [review]
The whole PR should be deleted.
Before submitting the next version ensure that these questions have been clarified:
- Does the GPL require that the complete code has to be GPL'ed, too (legal approval needed)?
- All files need a proper license statement.
- No preconfigured users are used. Should be configurable.
- No binaries with license or any other keys are included.

**Pull request closed without merging**